docs/contributing/Build-XKit.md

```diff
@@ -12,117 +12,61 @@
 
 1. Run `gulp server` to start the resource server.  This task will automatically build the extension and theme files from source before the server starts.  See [`gulp server`](#gulp-server) for more information.
-1. Navigate to `https://localhost:31337` and create a security exception for the `localhost` domain.  The project uses self-signed SSL/TLS certificates that are untrusted by default in order to work around mixed-content warnings for websites like Tumblr that are served over HTTPS.
-1. Change [the relevant line](https://github.com/new-xkit/XKit/blob/db88af1f6d232a4f3e8ba4626f28a0d64240e2a0/xkit.js#L233) with the URL string in `xkit.js` to `https://localhost:31337/Extensions/dist/` to point XKit at `localhost`.
-  - **Note**: Ensure the above URL is copied exactly.  Malformed URLs will result in XKit falling back to the old servers.
-1. Build the XKit extension from source with `gulp build:PLATFORM`, where `PLATFORM` is one of the supported platforms.  See [`gulp build`](#gulp-build) for more information.
-1. Reload the XKit extension in the browser under test:
+2. Navigate to `https://localhost:31337` and create a security exception for the `localhost` domain.  The project uses self-signed SSL/TLS certificates that are untrusted by default in order to work around mixed-content warnings for websites like Tumblr that are served over HTTPS.
+3. Change [the relevant line](https://github.com/new-xkit/XKit/blob/db88af1f6d232a4f3e8ba4626f28a0d64240e2a0/xkit.js#L233) with the URL string in `xkit.js` to `https://localhost:31337/Extensions/dist/` to point XKit at `localhost`.
+4. Reload the XKit extension in the browser under test:
   - Chrome: [reload the unpacked extension](https://developer.chrome.com/extensions/getstarted#unpacked)
-  - Firefox: [reload the temporary add-on](https://developer.mozilla.org/en-US/docs/Tools/about:debugging#Loading_a_temporary_add-on)
-1. Open the XKit settings menu and navigate to Other > Update All and click "Update all my extensions".
+  - Firefox: [reload the temporary add-on](https://developer.mozilla.org/en-US/docs/Tools/about:debugging#Extensions)
+5. Open the XKit settings menu and navigate to Other > Update All and click "Update all my extensions".
 
 > **Note**: changes to extension and theme files are not automatically propagated to the XKit extension in the browser.  Each time changes are made, XKit must be force-updated through "Update all my extensions" before the changes will be reflected.
 
-## Gulp Tasks:
-
-#### `gulp` (default)
-
-The default task.
-
-See also: [`gulp lint`](#gulp-lint).
-
-#### `gulp watch`
-
-Watches filesystem and lints files on change.
-
-See also: [`gulp lint:scripts`](#gulp-lintscripts), [`gulp lint:css`](#gulp-lintcss).
+## Scripts:
 
-#### `gulp clean`
+#### `npm test`
 
-Top-level cleaning task.
+Shortcut for [`gulp lint:scripts`](#gulp-lintscripts).
 
-See also: [`gulp clean:build`](#gulp-cleanbuild).
+#### `npm run-script dev`
```

Comment:
Is there a reason to do run-script instead of run? It seems a bit verbose

Reply:
i wasn't aware run was an alias for run-script, i'll tidy that up now